src/main/java/net/earthcomputer/clientcommands/command/VillagerCommand.java

```diff
@@ -0,0 +1,325 @@
+package net.earthcomputer.clientcommands.command;
+
+import com.mojang.brigadier.Command;
+import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.exceptions.CommandSyntaxException;
+import com.mojang.brigadier.exceptions.Dynamic2CommandExceptionType;
+import com.mojang.brigadier.exceptions.SimpleCommandExceptionType;
+import net.earthcomputer.clientcommands.Configs;
+import net.earthcomputer.clientcommands.command.arguments.ClientItemPredicateArgument;
+import net.earthcomputer.clientcommands.command.arguments.ItemAndEnchantmentsPredicateArgument;
+import net.earthcomputer.clientcommands.command.arguments.WithStringArgument;
+import net.earthcomputer.clientcommands.features.VillagerCracker;
+import net.earthcomputer.clientcommands.features.VillagerRngSimulator;
+import net.earthcomputer.clientcommands.task.SimpleTask;
+import net.earthcomputer.clientcommands.task.TaskManager;
+import net.earthcomputer.clientcommands.util.CUtil;
+import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
+import net.minecraft.ChatFormatting;
+import net.minecraft.advancements.critereon.MinMaxBounds;
+import net.minecraft.commands.CommandBuildContext;
+import net.minecraft.core.BlockPos;
+import net.minecraft.core.GlobalPos;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceKey;
+import net.minecraft.tags.EnchantmentTags;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.entity.npc.Villager;
+import net.minecraft.world.entity.npc.VillagerProfession;
+import net.minecraft.world.entity.npc.VillagerTrades;
+import net.minecraft.world.item.Item;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.TooltipFlag;
+import net.minecraft.world.level.Level;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.mojang.brigadier.arguments.IntegerArgumentType.*;
+import static dev.xpple.clientarguments.arguments.CBlockPosArgument.*;
+import static dev.xpple.clientarguments.arguments.CEntityArgument.*;
+import static dev.xpple.clientarguments.arguments.CRangeArgument.*;
+import static net.earthcomputer.clientcommands.command.ClientCommandHelper.*;
+import static net.earthcomputer.clientcommands.command.arguments.ClientItemPredicateArgument.*;
+import static net.earthcomputer.clientcommands.command.arguments.ItemAndEnchantmentsPredicateArgument.*;
+import static net.earthcomputer.clientcommands.command.arguments.WithStringArgument.*;
+import static net.fabricmc.fabric.api.client.command.v2.ClientCommandManager.*;
+
+public class VillagerCommand {
```

Comment:
Bit late to it, but I feel like this could command could have a better name.

Reply:
any suggestions? we have `cenchant` for enchantment manipulation, `cchorus` for chorus manipulation, I think it fits rather well with those

Reply:
Fair enough.

Reply:
`/ctrade` ?

Reply:
is there a need to rename?

Reply:
I wouldn't say so